### ISSUE TITLE
Feature Request: Indicate Player's Involvement on Games & Notifications

### ISSUE THREAD

**Issue by Wh1rledPeas** [mlapeter/the100reactapp#64]
In the attached screenshots I am already joined to the top two games in the list. It would be nice to have some type of indicator, showing whether I'm joined already. As it is, I have to tap on each of them to know for sure.

![screenshot_20181120-081459](https://user-images.githubusercontent.com/3685437/48776402-c5f52c80-ec9d-11e8-84ec-27702edb8302.png)

![screenshot_20181120-082019](https://user-images.githubusercontent.com/3685437/48776426-d60d0c00-ec9d-11e8-954b-7cfbe6e936bc.png)

**Comment by mlapeter** [comment]
Thanks for the idea! I should be able to implement for the actual gaming session (second screenshot) but may be trickier for notification as those are more basic info (user id, avatar, notification message etc) and don't include detailed info about the gaming session.

**Comment by Wh1rledPeas** [comment]
I see you made some updates and changed the background to green. I notice if I'm signed up as reserve though, it doesn't show anything. Perhaps use a muted color for those?

**Comment by mlapeter** [comment]
Good catch! I'll see about adding a similar green for those sessions too.
